examples/flows/transfer-mv-yaml/definition.yaml

```diff
@@ -0,0 +1,81 @@
+Comment: >-
```

Comment:
It's extra nice to see the right chomping bit, I think using it in the example will save a lot of users headaches.